rust/extractor/src/config.rs

```diff
@@ -32,6 +32,7 @@ pub struct Config {
     pub scratch_dir: PathBuf,
     pub trap_dir: PathBuf,
     pub source_archive_dir: PathBuf,
+    pub extract_dependencies: bool,
```

Comment:
It's already a CodeQL extractor option which can be passed to `codeql database create` . Not sure if there is a point having all those flags repeated in the extractor binary too.

Reply:
from experience it can be nice to try out the extractor in isolation from `codeql`, and without writing all that screaming snake case in the env :D. Also, `qltest.sh` allows to inject extractor flags in the test sources via comments, so it can be useful to have CLI flags. But we can also inject env variables there, so this is not relevant.